templates/rust-starter/README.md

```diff
@@ -41,6 +41,28 @@ Search this template for the string `TODO`, and make the necessary changes to im
  * Reference documentation for our Rust crates is available at [docs.rs], including the [RISC Zero zkVM crate](https://docs.rs/risc0-zkvm), the [cargo risczero crate](https://docs.rs/cargo-risczero), the [RISC Zero build crate](https://docs.rs/risc0-build), and others (the full list is available at [https://github.com/risc0/risc0/blob/main/README.md]).
  * Our [main repository](https://www.github.com/risc0/risc0).
 
+## Directory Structure
+It is possible to organize the files for these components in various ways. 
+However, in the [`cargo risczero`](https://docs.rs/cargo-risczero/0.16/cargo_risczero/) starter template we use a standard directory structure for zkVM applications, and we recommend you use this structure as well. 
```

Comment:
Let's take the opportunity to be a little less prescriptive (in large projects especially there may be good project-specific reasons to organize differently).
```suggestion
However, in the [`cargo risczero`](https://docs.rs/cargo-risczero/0.16/cargo_risczero/) starter template we use a standard directory structure for zkVM applications, which we think is a good starting point for your applications.
```